examples/README.md

```diff
@@ -7,4 +7,135 @@ Each directory contains one example.
 - `rust-**` denotes that the userspace program is written in Rust. 
 - Others means that the userspace program is written in `C/C++`
 
-Go examples requires `tinygo`, rust examples requires `cargo`.
\ No newline at end of file
+Go examples requires `tinygo`, rust examples requires `cargo`.
+
+### C example: [Bootstrap](examples/bootstrap)
+
+`bootstrap` is an example of a simple (but realistic) BPF application. It
+tracks process starts (`exec()` family of syscalls, to be precise) and exits
+and emits data about filename, PID and parent PID, as well as exit status and
+duration of the process life. With `-d <min-duration-ms>` you can specify
+minimum duration of the process to log. In such mode process start
+(technically, `exec()`) events are not output (see example output below).
+
+`bootstrap` was created in the similar spirit as
+[libbpf-tools](https://github.com/iovisor/bcc/tree/master/libbpf-tools) from
+BCC package, but is designed to be more stand-alone and with simpler Makefile
+to simplify adoption to user's particular needs. It demonstrates the use of
+typical BPF features:
+
+- cooperating BPF programs (tracepoint handlers for process `exec` and `exit`
+    events, in this particular case);
+- BPF map for maintaining the state;
+- BPF ring buffer for sending data to user-space;
+- global variables for application behavior parameterization.
+- it utilizes BPF CO-RE and vmlinux.h to read extra process information from
+    kernel's `struct task_struct`.
+
+Here's an example output:
+
+```console
+$ sudo ./wasm-bpf bootstrap.wasm -h
+BPF bootstrap demo application.
+
+It traces process start and exits and shows associated 
+information (filename, process duration, PID and PPID, etc).
+
+USAGE: ./bootstrap [-d <min-duration-ms>] -v
+$ sudo ./wasm-bpf bootstrap.wasm
+TIME     EVENT COMM             PID     PPID    FILENAME/EXIT CODE
+18:57:58 EXEC  sed              74911   74910   /usr/bin/sed
+18:57:58 EXIT  sed              74911   74910   [0] (2ms)
+18:57:58 EXIT  cat              74912   74910   [0] (0ms)
+18:57:58 EXEC  cat              74913   74910   /usr/bin/cat
+18:57:59 EXIT  cat              74913   74910   [0] (0ms)
+18:57:59 EXEC  cat              74914   74910   /usr/bin/cat
+18:57:59 EXIT  cat              74914   74910   [0] (0ms)
+18:57:59 EXEC  cat              74915   74910   /usr/bin/cat
+18:57:59 EXIT  cat              74915   74910   [0] (1ms)
+18:57:59 EXEC  sleep            74916   74910   /usr/bin/sleep
+```
+
+See [examples/bootstrap](examples/bootstrap) for more details.
+
+### Rust example: [Bootstrap](examples/rust-bootstrap)
+
+similar to C bootstrap, but written in Rust.
+
+See [examples/rust-bootstrap](examples/rust-bootstrap) for more details.
+
+### C example: [runqlat](examples/runqlat)
+
+This program summarizes scheduler run queue latency as a histogram, showing
+how long tasks spent waiting their turn to run on-CPU.
+
+```console
+$ sudo ./wasm-bpf runqlat.wasm -h
```

Comment:
```suggestion
$ sudo ./wasm-bpf -h
```

Reply:
if support hashbang？

```suggestion
$ sudo ./runqlat.bpf -h
```

Reply:
I misunderstood, this help information is from wasm.
No need to change.